addons/io/org.openhab.io.hueemulation/README.md

```diff
@@ -34,3 +34,11 @@ Switch  TestSwitch2     "Bathroom" [ "Lighting" ]
 Dimmer  TestDimmer3     "Hallway" [ "Lighting" ]
 Number  TestNumber4     "Temperature Set Point" [ "TargetTemperature" ]
 ```
+If you have an item with a channel or binding attached the tag needs to be applied before the channel
+```
```

Comment:
Remove lines 38-41.

Reply:
I can't seem to reload the file but here is a simpler example

`Switch TVSwitch_Switch "TV Switch" (LivingRoom) { channel="tplinksmarthome:hs110:TV:switch" }`
becomes
`Switch TVSwitch_Switch "TV Switch" (LivingRoom) [ "Switchable" ]{ channel="tplinksmarthome:hs110:TV:switch" }`

Reply:
@pmknowles Ping. Any news on this?